src/_tsd.d.ts

```diff
@@ -1,3 +1,3 @@
 /// <reference path="../typings/knockout/knockout.d.ts" />
-/// <reference path="../node_modules/definition-header/dist/index.d.ts" />
+/// <reference path="../node_modules/tsd/node_modules/definition-header/dist/index.d.ts" />
```

Comment:
Yes, that would be nice.
El 30/12/2015 12:40, "lucasasecas" <notifications@github.com> escribió:

> In src/_tsd.d.ts
> <https://github.com/MakingSense/board-poc/pull/2#discussion_r48610454>:
>
> > @@ -1,3 +1,3 @@
> >  /// <reference path="../typings/knockout/knockout.d.ts" />
> > -/// <reference path="../node_modules/definition-header/dist/index.d.ts" />
> > +/// <reference path="../node_modules/tsd/node_modules/definition-header/dist/index.d.ts" />
>
> This change is part of #1
> <https://github.com/MakingSense/board-poc/pull/1> pull request, right?
>
> Yes, I made the mistake to push with an old commit.
> I have to undo this commit and re-push right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MakingSense/board-poc/pull/2/files#r48610454>.
>